### DISCUSSION TITLE
Account creation for cross-device

### DISCUSSION THREAD

**Discussion by RafaDarque** [vkbo/novelWriter#1204] [Ideas]
Hello! Would be amazing to have the feature to create an account in the software, so after we could go to another pc (for example) and continue our work. Thank you for your consideration :)

**Comment by vkbo** [comment]
I'm not sure I follow. Do you mean adding a cloud service? That is expensive and there is virtually no money in this.

novelWriter has a project format that is well suited for standard file syncing tools as long as they don't use a virtual file system like Google does. I use novelWriter on several computers myself, and there is no problem.

**Comment by Valdus92** [comment]
What standard file syncing would you recommend. I tried a usb and it gets tricky on my chromebook with the Linux drive separated.

**Reply by vkbo** [reply]
Any sync tool that doesn't use a virtual file system (like Google Drive does on Linux/Gnome) should work. Personally, I use Syncthing.

As long as you let the files sync properly before you suspend/power down, there should be no issue. Keep in mind that the deletion of the lock file needs to propagate too, which is the last change when you close the project or novelWriter. All files in a novelWriter project are relatively small text files (some in XML and JSON). Changes should sync pretty fast with any sync tool.

**Reply by johnblommers** [reply]
Let me second the vote for Syncthing. It is awesome and works really well so long as your file system has a reasonable amount of free space left. It even works when you're away from your home LAN via the global discovery feature. And because of end-to-end encryption it's totally secure.
